README.md

```diff
@@ -50,12 +61,12 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
 Your Answer:
 
 ```text
-___: Defines the distance between an element's border and adjacent elements' borders.
-___: Inserts a "wall" around an element.
-___: Defines the distance between an element's content and its border.
+__margin__: Defines the distance between an element's border and adjacent elements' borders.
+__outline__: Inserts a "wall" around an element.
+__padding__: Defines the distance between an element's content and its border.
 ___: Defines the width of an element.
 ___: Overlays a "wall" on top of an element.
-___: Defines the distance between the center of an element and the center of the adjacent element.
+__border__: Defines the distance between the center of an element and the center of the adjacent element.
 ```
```

Comment:
Close! Answers below. Let me know if I can clarify any of these. Border vs. outline can be confusing, but to be honest I don't think the difference will matter much in the real world unless you go pretty deep into CSS.

```text
margin: Defines the distance between an element's border and adjacent elements' borders.
border: Inserts a "wall" around an element.
padding: Defines the distance between an element's content and its border.
___: Defines the width of an element.
outline: Overlays a "wall" on top of an element.
___: Defines the distance between the center of an element and the center of the adjacent element.
```